addons/account_edi_ubl_cii/models/res_partner.py

```diff
@@ -125,6 +125,16 @@ class ResPartner(models.Model):
         ]
     )
 
+    @api.constrains('peppol_eas')
+    def _check_peppol_eas(self):
+        for partner in self:
+            if partner.peppol_eas in ('0212', '0213'):
```

Comment:
funny how this is not mentioned in https://docs.peppol.eu/edelivery/codelists/changelog.html but it is by the Finish Authority :sweat:, so we can just rely on the docs.peppol.eu website...